dsa-problems/leetcode-problems/0500-0599.md

```diff
@@ -170,7 +170,7 @@ export const problems =[
     "problemName": "528. Random Pick with Weight",
     "difficulty": "Medium",
     "leetCodeLink": "https://leetcode.com/problems/random-pick-with-weight",
-    "solutionLink": "#"
+    "solutionLink": "/dsa-solutions/lc-solutions/0500-0599/0528-Random-Pick-with-Weight"
```

Comment:
Please remember the following text:
Replace `0528-Random-Pick-with-Weight` with `random-pick-with-weight`.